articles/azure-app-configuration/enable-dynamic-configuration-dotnet-core-push-refresh.md

```diff
@@ -175,6 +175,49 @@ A random delay is added before the cached value is marked as dirty to reduce pot
 > [!NOTE]
 > To reduce the number of requests to App Configuration when using push refresh, it is important to call `SetCacheExpiration(TimeSpan cacheExpiration)` with an appropriate value of `cacheExpiration` parameter. This controls the cache expiration time for pull refresh and can be used as a safety net in case there is an issue with the Event subscription or the Service Bus subscription. The recommended value is `TimeSpan.FromDays(30)`.
 
+## Optional: Incorporate Synchronization tokens for real-time consistency
+
+> [!NOTE]
+> Support for using synchronization-tokens in push refresh workflow is available in .NET configuration provider version 5.0.0-preview (or later). 
```

Comment:
Nit: remove hyphen from `synchronization-tokens` for consistent usage of the term.